lib/web/command.go

```diff
@@ -674,7 +674,12 @@ func (t *commandHandler) streamOutput(ctx context.Context, tc *client.TeleportCl
 		return
 	}
 
-	if err := t.stream.SendCloseMessage(); err != nil {
+	sessionMetadataPayload, err := json.Marshal(siteSessionGenerateResponse{Session: t.sessionData})
```

Comment:
I'm not sure if sending all session data is a good idea. We need only node ID, and session data may contain some stuff that we may not want to share (if not now it may in the future).